examples/with-nextjs-and-clerk-auth/src/components/Layout.tsx

```diff
@@ -20,12 +20,8 @@ export const Layout = ({ children }: { children: ReactNode }) => {
       display="flex"
       className={`Monite-Page-${currentPath}`}
       sx={{
-        position: 'absolute',
-        left: '0',
-        right: '0',
-        top: '0',
-        bottom: '0',
-        overflow: 'hidden',
+        width: '100vw',
```

Comment:
This is not really the [same](https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser). We could use `lvh` instead, but firefox doesn't support it yet.

Reply:
I removed absolute positioning to preserve consistency across SDK. Since our UI isn't mobile-friendly yet, I believe we can postpone it.